tensorflow/compiler/xla/service/gpu/hlo_to_ir_bindings.cc

```diff
@@ -119,7 +119,9 @@ void HloToIrBindings::EmitBasePointersForHlos(
           if (slice.allocation()->is_thread_local()) {
             llvm::Type* pointee_type =
                 llvm_ir::ShapeToIrType(non_io_hlo->shape(), module_);
-            BindHloToIrValue(*non_io_hlo, b_->CreateAlloca(pointee_type),
+            BindHloToIrValue(*non_io_hlo,
+                             llvm_ir::EmitAllocaAtFunctionEntry(
+                                 pointee_type, absl::string_view(), b_),
```

Comment:
`absl::string_view()` -> `""`?

Reply:
`absl::string_view()` is intentional.

It will result in passing a `nullptr` to the `createAlloca` here (https://github.com/ROCmSoftwarePlatform/tensorflow-upstream/blob/develop-upstream/tensorflow/compiler/xla/service/llvm_ir/llvm_util.cc#L293). 

That will match the current behaviour (i.e. passing a `nullptr` as the name). 

Think `absl::string_view()` and `absl::string_view("")` are equivalent.

https://github.com/abseil/abseil-cpp/blob/master/absl/strings/string_view.h#L139-L150

Reply:
yes, so that's why I say switch to `""`, or better yet:

```
/*name=*/""
```

this makes the code easier to follow.